bukkit/src/main/java/dev/aurelium/auraskills/bukkit/requirement/RequirementManager.java

```diff
@@ -144,6 +230,14 @@ public List<BlockRequirement> getBlocks() {
         return blockRequirements;
     }
 
+    public LootRequirement getLootRequirementByID(NamespacedId id) {
```

Comment:
Rename the "ID" part to "Id". The convention in this project is that only the first letter of any acronym is capitalized.